medicines/doc-index-updater/scripts/api.scala

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
without a readme entry on how to use this, it probably doesn't belong in the repo ?

Reply:
Yeah, some docs on gatling even if they're brief would be handy

Reply:
done